assets/src/edit-story/components/form/dropDown/list.js

```diff
@@ -99,6 +99,12 @@ function DropDownList({
     [focusedValue, options]
   );
 
+  useEffect(() => {
+    // Hide dropdown menu on scroll.
```

Comment:
_scroll_ or _wheel_?

Per [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/wheel_event):

>  Don't confuse the `wheel` event with the `scroll` event. The default action of a `wheel` event is implementation-specific, and doesn't necessarily dispatch a `scroll` event.

Also, isn't that event fired dozens of times? Should the callback be throttled?

Reply:
Does this only happen on wheel? Feel like it would happen on scroll as well.

Reply:
I just confirmed, that wheel can not be used. The page up and page down buttons also cause the issue you are trying to fix here.

Reply:
Using `scroll` instead.

Set `options` to `true` so "the callback will only be invoked once after which the event listener will be removed".